udata_front/theme/gouvfr/assets/js/components/AdminTable/AdminDiscussionsTable/AdminDiscussionsTable.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <AdminTable class="fr-table--layout-fixed" :loading>
+    <thead>
+      <AdminTableTh
+        @sort="(direction: SortDirection) => $emit('sort', 'title', direction)"
+        :sorted="sorted('title')"
+        scope="col"
+      >
+        {{ t("Discussion") }}
+      </AdminTableTh>
+      <AdminTableTh scope="col">
+        {{ t("Status") }}
+      </AdminTableTh>
+      <AdminTableTh
+        @sort="(direction: SortDirection) => $emit('sort', 'created', direction)"
+        :sorted="sorted('created')"
+        scope="col"
+      >
+        {{ t("Created at") }}
+      </AdminTableTh>
+      <AdminTableTh
+        @sort="(direction: SortDirection) => $emit('sort', 'closed', direction)"
+        :sorted="sorted('closed')"
+        scope="col"
+      >
+        {{ t("Closed at") }}
+      </AdminTableTh>
+      <AdminTableTh scope="col" style="width: 25%;">
+        {{ t("URL") }}
+      </AdminTableTh>
+    </thead>
+    <tbody>
+      <tr v-for="discussion in discussions">
+        <td>
+          <p class="fr-text--bold">
+            <TextClamp
+              :text="discussion.title"
+              :auto-resize="true"
+              :max-lines="1"
+            />
+          </p>
+          <p v-if="subjects[discussion.subject.id]">
+            <a
+              class="fr-link inline-flex"
+              :href="subjects[discussion.subject.id].page"
+            >
+              <Vicon :height="12" class="self-center" :name="subjects[discussion.subject.id].icon"/>
+              <TextClamp
+                class="overflow-wrap-anywhere"
+                :text="subjects[discussion.subject.id].title"
+                :auto-resize="true"
+                :max-lines="1"
+              />
+            </a>
+          </p>
+        </td>
+        <td>
+          <AdminBadge :type="getStatus(discussion).type">
+            {{ getStatus(discussion).label }}
+          </AdminBadge>
+        </td>
+        <td>{{ formatDate(discussion.created) }}</td>
+        <td>
+          <template v-if="discussion.closed">{{ formatDate(discussion.closed) }}</template></td>
+        <td>
+          <a
+            v-if="subjects[discussion.subject.id]"
+            class="fr-link inline-flex"
+            :href="getDiscussionUrl(discussion.id, subjects[discussion.subject.id])"
+            target="_blank"
```

Comment:
Should the `new tab` icon be underlined?

Reply:
This is the case in my tests on both Firefox and Chrome. We can show our own icon instead of the DSFR magic if it's not the case everywhere

Reply:
Note : The icon is not underlined on Figma but it's way easier and accessible to have it underlined